product/timelines/miq_reports/tl_events_daily.yaml

```diff
@@ -36,6 +36,7 @@ cols:
 - container_namespace
 - container_name
 - container_replicator_name
+- middleware_name
```

Comment:
what field/entity does this represent? middleware_name sounds a bit too generic
(compared, for example to container_replicator_name)